lib/segment/src/types.rs

```diff
@@ -284,9 +284,9 @@ pub struct QuantizationSearchParams {
 
     /// If true, use original vectors to re-score top-k results.
     /// Might require more time in case if original vectors are stored on disk.
-    /// Default is false.
-    #[serde(default = "default_quantization_rescore_value")]
-    pub rescore: bool,
+    /// If not set, qdrant decides automatically apply rescoring or not.
```

Comment:
```suggestion
    ///
    /// If not set, qdrant decides automatically to re-score or not, depending on quantization settings.
```

Should we include the example of binary quantization having a side-effect on score?